docker/Dockerfile.release

```diff
@@ -0,0 +1,108 @@
+# NOTICE: To keep consistency across this docker file, scripts/setup_linux.sh
+# and scripts/setup_macos.sh, if there's any changes applied to this file,
+# specially regarding the installation of dependencies, apply those same
+# changes to the mentioned files.
+FROM ubuntu:16.04
```

Comment:
This image uses a fake display, meaning I can't ever see the training visualization. Why not have fake display, docker-opengl, and nvidia-docker versions of this?

The canonical one would be docker-opengl, since that will run on any machine interactively.

Reply:
I could generate images for the fake display (I named it headless) and nvidia versions. However, I couldn't find/create a proper opengl image that works with ubuntu 16.04. I'm taking a look at which packages the nvidia-opengl uses to copy the opengl parts only, but I can submit this version in a next PR.